index.html

```diff
@@ -53,12 +53,14 @@
                             <li><a href="#" onclick="setFrequency(64);">64 Hz</a></li>
                             <li><a href="#" onclick="setFrequency(128);">128 Hz</a></li>
                             <li><a href="#" onclick="setFrequency(256);">256 Hz</a></li>
+                            <li><a href="#" onclick="setFrequency(1024);">1024 Hz</a></li>
```

Comment:
See https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setInterval

> **delay**
> The time, in milliseconds (thousandths of a second), the timer should delay in between executions of the specified function or code. If this parameter is less than 10, a value of 10 is used. Note that the actual delay may be longer; see "Reasons for delays longer than specified" in WindowOrWorkerGlobalScope.setTimeout() for examples.

Setting a timeout of 1024 would mean that the `setInterval` call will receive a delay of `1000/1024`, which has no difference when compared to `1000/256`.